packages/wallets/src/features/cashier/modules/Transfer/components/TransferReceipt/__tests__/TransferReceipt.spec.tsx

```diff
@@ -0,0 +1,139 @@
+/* eslint-disable camelcase */
+import React, { PropsWithChildren } from 'react';
+import { APIProvider } from '@deriv/api-v2';
+import { render, screen, within } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import WalletsAuthProvider from '../../../../../../../AuthProvider';
+import { TransferProvider } from '../../../provider';
+import { TAccount } from '../../../types';
+import TransferReceipt from '../TransferReceipt';
+
+const ACCOUNTS: NonNullable<TAccount>[] = [
+    {
+        account_category: 'wallet',
+        account_type: 'doughflow',
+        balance: '1000',
+        currency: 'USD',
+        currencyConfig: {
+            fractional_digits: 2,
+        },
+    },
+    {
+        account_category: 'wallet',
+        account_type: 'crypto',
+        balance: '0.1',
+        currency: 'BTC',
+        currencyConfig: {
+            fractional_digits: 8,
+        },
+    },
+    {
+        account_category: 'wallet',
+        account_type: 'crypto',
+        balance: '1',
+        currency: 'ETH',
+        currencyConfig: {
+            fractional_digits: 8,
+        },
+    },
+    {
+        account_category: 'wallet',
+        account_type: 'crypto',
+        balance: '1',
+        currency: 'LTC',
+        currencyConfig: {
+            fractional_digits: 8,
+        },
+    },
+] as NonNullable<TAccount>[];
```

Comment:
Why do you need to type it twice 🐱

Reply:
mann I just reused the mocks from another test file 🤣